pkg/api/controller_test.go

```diff
@@ -3651,27 +3652,44 @@ func TestLDAPClient(t *testing.T) {
 
 		// bad user credentials with anonymous authentication
 		lClient = &api.LDAPClient{
```

Comment:
What are these 2 cases testing differently? Was it a redundance in our test code?